ocfweb/account/templates/account/register/index.html

```diff
@@ -58,8 +58,7 @@
                         <div class="col-sm-7">
                             <p>Your account name must consist of between three and 16 lowercase letters (no spaces, numbers, underscores, or other symbols), and it <strong><strong>MUST</strong> be based on your real name or initials.</strong></p>
                             {{form.ocf_login_name|bootstrap}}
-                            <div class="alert alert-success" role="alert" id="valid_username_notif" style="display: none; padding: 7px"></div>
-                            <div class="alert alert-danger" role="alert" id="invalid_username_notif" style="display: none; padding: 7px"></div>
+                            <div class="alert" role="alert" id="username_feedback" style="display: none; padding: 7px"></div>
```

Comment:
It would be better if this CSS was in a stylesheet instead of inline here, but this is fine for now.